### PR TITLE
More details around the jtreg bug tag

### PR DESCRIPTION
**Pull request by JesperIRL**
Minor change to clarify what kind of issues to refer to with the jtreg bug tag.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Iris Clark](https://openjdk.org/census#iris) (@irisclark - **Reviewer**) ⚠️ Review applies to [feecd73e](https://git.openjdk.org/guide/pull/96/files/feecd73e0e4265203d7b0f76c89b57e56ef0b329)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - no project role) ⚠️ Review applies to [feecd73e](https://git.openjdk.org/guide/pull/96/files/feecd73e0e4265203d7b0f76c89b57e56ef0b329)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/guide pull/96/head:pull/96` \
`$ git checkout pull/96`

Update a local copy of the PR: \
`$ git checkout pull/96` \
`$ git pull https://git.openjdk.org/guide pull/96/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 96`

View PR using the GUI difftool: \
`$ git pr show -t 96`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/guide/pull/96.diff">https://git.openjdk.org/guide/pull/96.diff</a>

</details>

### CODE REVIEW

File: src/guide/testing-the-jdk.md [hunk 1]
```diff
@@ -48,13 +48,15 @@ public class TestXY {
...
-There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for.
+There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for. The bug id is written without the `JDK-` prefix.
 
 ~~~
-@bug 7000001
+@bug 8272373
 ~~~
 
-Or you can specify a number of requirements that must be fulfilled for jtreg to execute the test.
+You can add several bug ids in the same `@bug` tag, separated by a single space. These bug ids should refer to bugs that are reproduced by this test. JBS issues that track changes to the test itself are not listed here.
```
Comment (stuart-marks) [inline src/guide/testing-the-jdk.md:57]: "Reproduced" doesn't sound quite right. I think a bug id should be listed here if this test is verifying the _fix_ for that bug.

Comment (JesperIRL) [reply src/guide/testing-the-jdk.md]: Right. Tried to rephrase it to say that.

**Review by irisclark — APPROVED** [review]

File: src/guide/testing-the-jdk.md [hunk 2]
```diff
@@ -48,13 +48,15 @@ public class TestXY {
...
-There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for.
+There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for. The bug id is written without the `JDK-` prefix.
 
 ~~~
-@bug 7000001
+@bug 8272373
 ~~~
 
-Or you can specify a number of requirements that must be fulfilled for jtreg to execute the test.
+You can add several bug ids in the same `@bug` tag, separated by a single space. The bug ids listed here should refer to bugs for which this test is verifying the fix. JBS issues that track changes to the test itself are not listed here.
```
Comment (aivanov-jdk) [inline src/guide/testing-the-jdk.md:57]: Is the comma necessary before “separated”?

**Review by aivanov-jdk — APPROVED** [review]

File: src/guide/testing-the-jdk.md (hunk 2 above)
Comment (JesperIRL) [reply src/guide/testing-the-jdk.md:57]: That's a good question. I'll defer to someone who is native English speaker to determine that.

Comment (msheppar) [reply src/guide/testing-the-jdk.md:57]: In think it is ok, as it emphasizes the first part of the statement. A slighly more verbose form would be:
You can add several bug ids in the same `@bug` tag, each bug id is separated by a single space.
This in turn could be structured as two separate sentences. 
You can add several bug ids in the same `@bug` tag. Each bug id is separated by a single space.

File: src/guide/testing-the-jdk.md (hunk 1 above)
Comment (msheppar) [reply src/guide/testing-the-jdk.md]: perhaps?
These bug ids refer to product bugs (, not test bugs), for which a fix is verified by this test. JBS issues that track changes to the test itself are not listed here.

File: src/guide/testing-the-jdk.md [hunk 3]
```diff
@@ -48,13 +48,15 @@ public class TestXY {
...
-There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for.
+There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for. The bug id is written without the `JDK-` prefix.
 
 ~~~
-@bug 7000001
+@bug 8272373
 ~~~
 
-Or you can specify a number of requirements that must be fulfilled for jtreg to execute the test.
+You can add several bug ids in the same `@bug` tag, separated by a single space. These bug ids refer to product bugs for which a fix is verified by this test. JBS issues that track changes to the test itself are not listed here.
```
Comment (LanceAndersen) [inline src/guide/testing-the-jdk.md:57]: Should there be a mention that if there is a fix applied to the test which is not related to fixing an implementation issue, then the developer should not add the bug number to the test via `@bug`

Comment (aivanov-jdk) [reply src/guide/testing-the-jdk.md:57]: > Should there be a mention that if there is a fix applied to the test which is not related to fixing an implementation issue, then the developer should not add the bug number to the test via `@bug`

Doesn't the sentence _“JBS issues that track changes to the test itself are not listed here,”_ state exactly this?
